client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -99,7 +85,19 @@ export const HoldList = ( {
 				{ count: holds.length }
 			) } />
 			<Card className="eligibility-warnings__hold-list">
-				{ map( holds, hold =>
+				{ isPlaceholder &&
+					<div>
+						<div className="eligibility-warnings__hold">
+							<Gridicon icon="notice-outline" size={ 24 } />
+							<div className="eligibility-warnings__message"></div>
+						</div>
+						<div className="eligibility-warnings__hold">
+							<Gridicon icon="notice-outline" size={ 24 } />
+							<div className="eligibility-warnings__message"></div>
+						</div>
+					</div>
+				}
```

Comment:
We do use it here! https://github.com/Automattic/wp-calypso/blob/master/client/blocks/eligibility-warnings/style.scss#L7

Though, as long as the logic is ok, I'd defer style updates to another PR to keep this one as focused to the copy as possible.